src/main/java/fanny/task/Event.java

```diff
@@ -0,0 +1,59 @@
+package fanny.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents an event task with a specific start and end time.
+ * Inherits from the {@link Task} class.
+ */
+public class Event extends Task {
+
+    /** The start time of the event. */
+    protected LocalDateTime from;
```

Comment:
Perhaps startDate/startTime and endDate/endTime would be more accurate?

Reply:
Thank you for your comment. I think startTime and endTime is indeed more accurate.